docs/index.rst

```diff
@@ -19,6 +19,8 @@ user-defined types among other features.
 With Solidity you can create contracts for uses such as voting, crowdfunding, blind auctions,
 and multi-signature wallets.
 
+When deploying contracts, you should use the latest released version of Solidity because bug fixes are rarely, if ever, back-ported to older versions.
```

Comment:
Perhaps also mention that we are making breaking changes often and so on like in the issue description?

Reply:
Updated in https://github.com/fulldecent/solidity/commit/fb7b55dac83b859df3c5ac3628eb9c878ae4fb1f

Reply:
Updated in https://github.com/fulldecent/solidity/commit/fb7b55dac83b859df3c5ac3628eb9c878ae4fb1f